packages/core/src/plugins/html-deserializer/utils/deserializeHtml.ts

```diff
@@ -18,6 +19,7 @@ export const deserializeHtml = <V extends Value>(
     stripWhitespace?: boolean;
   }
 ): EDescendant<V>[] => {
+  setStripWhitespace(stripWhitespace);
   // for serializer
   if (typeof element === 'string') {
     element = htmlStringToDOMNode(element, stripWhitespace);
```

Comment:
Here's the `stripWhitespace` option that we no longer need. You can remove this from `htmlStringToDOMNode` entirely.